next.config.js

```diff
@@ -1,4 +1,6 @@
-module.exports = {
+const withTM = require('next-transpile-modules')(['react-syntax-highlighter']); // pass the modules you would like to see transpiled
+
+module.exports = withTM({
```

Comment:
What does `withTM` mean?

Reply:
I used the example from their readme :) 
I believe means `with transpile modules` 

In the description i mentioned I had to add this to support ES6 in node_modules